### PR TITLE
ARGO-513 Add Project Admin and service admin roles

### PR DESCRIPTION
**Pull request by kaggis**
- Add Project Admin role to manage project resources/users 
- Add missing service admin role to manage project creation (and assign initial users)
- Add Init Projects template and tasks
- Add SERVICE project (for establishing service_admins and assigning them the service_admin role)
- Add ARGO project (default project for demos and tests)
- Update init users template to use project_uuid

### CODE REVIEW

**Comment by kaggis** [comment]
please review
/cc @themiszamani

File: roles/messaging_api/defaults/main.yml [hunk 1]
```diff
@@ -6,6 +6,21 @@ argo_msg_store_db: 'argo_msg'
...
 
 messaging_api_users:
+  -  username: default_service_admin
+     project: ARGO
+     key: SERVICEADMINTOKEN
+     role: service_admin
+     email: service_admin@argo.example.foo
+  -  username: default_project_admin
+     project: ARGO
+     key: PROJETADMINTOKEN
```
Comment (themiszamani) [inline roles/messaging_api/defaults/main.yml]: please use real tokens .

File: roles/messaging_api/defaults/main.yml [hunk 2]
```diff
@@ -6,6 +6,21 @@ argo_msg_store_db: 'argo_msg'
...
+     role: project_admin
+     email: project_admin@argo.example.foo
+  -  username: default_viewer
+     project: ARGO
+     key: VIEWERTOKEN
+     role: viewer
+     email: viewer@argo.example.foo
   -  username: default_admin
      project: ARGO
      key: ADMINTOKEN
```
Comment (themiszamani) [inline roles/messaging_api/defaults/main.yml]: the same here

**Comment by kaggis** [comment]
- Tokens updated

please review
/cc @themiszamani

**Pull request merged**